dags/iop/repository.py

```diff
@@ -16,10 +16,7 @@ def __init__(self) -> None:
         self.s3 = S3Service(os.getenv("IOP_BUCKET_NAME", "iop"))
 
     def get_all_raw_filenames(self):
-        return [
-            os.path.basename(f.key)
-            for f in self.s3.objects.filter(Prefix=self.ZIPED_DIR).all()
-        ]
+        return [f.key for f in self.s3.objects.filter(Prefix=self.ZIPED_DIR).all()]
```

Comment:
why do we need to change that?

Reply:
f is s3.ObjectSummary.
This place actually needs more improvement. We want to return zip file paths, without "raw/". So we need to do the same is in [Springer](https://github.com/cern-sis/workflows/blob/c30a6dd0a12aa5a360d135e49ff67caf2760ed5e/dags/springer/repository.py#L17-L21)